src/plugins/theme/borderRadius.js

```diff
@@ -4,7 +4,8 @@
 module.exports = function () {
   return {
     'none': '0',
-    DEFAULT: '0.3rem', // form input border-radius
```

Comment:
No more default? Why is this being removed?

Reply:
I'm on the fence about this. It seems the 0.3rem is only used for input fields, so I feel like calling it our default might be a little misleading, but I could be convinced otherwise 😄

Reply:
https://tailwindcss.com/docs/border-radius#border-radiuses

I still think we need to provide a `DEFAULT` option to tailwind. It is used in the `su-rounded` class. You can choose the sensible default that you like but I think we should still have one.

Reply:
Sounds good - will have it =0.3rem then

Reply:
Thinking about this... I'm not completely sure about the 7px radius on the local footer login button. Will talk to @kerri-augenstein and see if I can convince her to maybe have this also equal to 0.3rem 😄  - but that's for later.